R/pkg/R/functions.R

```diff
@@ -3006,6 +3008,27 @@ setMethod("array_contains",
             column(jc)
           })
 
+#' @details
+#' \code{array_join}: Concatenates the elements of column using the delimiter.
+#' Null values are replaced with nullReplacement if set, otherwise they are ignored.
+#'
+#' @param delimiter character(s) to use to concatenate the elements of column.
```

Comment:
I didn't check scala - what's "(s)" here in "character(s)" mean? I ask because "character" refers to the type in R

Reply:
@felixcheung scala doesn't have a doc for param delimiter. I added this myself. What I am trying to say is "one or more characters". I will change to "a character string" so it will be
```
@param delimiter a character string to use to concatenate the elements of column.
```
Does this look ok to you?